canonical-websites/snapcraft.io
# Improve code styling on descriptions

Fixes https://github.com/canonical-websites/snapcraft.io/issues/1460
Simple improvement for code and code block styling within descriptions

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/lukewh-test (which uses the copy from /htop as it *could* appear)
